L1Trigger/TrackerTFP/interface/State.h

```diff
@@ -0,0 +1,166 @@
+#ifndef L1Trigger_TrackerTFP_State_h
+#define L1Trigger_TrackerTFP_State_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
+#include <vector>
+#include <numeric>
+
```

Comment:
All classes should have initial comment explaining what they are.

Reply:
done